bindings/go/src/fdb/database.go

```diff
@@ -96,7 +96,7 @@ func retryable(wrapped func() (interface{}, error), onError func(Error) FutureNi
 			e = onError(ep).Get()
 		}
 
-		/* If OnError returns an error, then it's not
+		/* If OnError returns an error, then it's not       */
```

Comment:
I'm inclined to say we should also use `//` for this comment, even though there are other cases (mostly just in this file) where we use `/* */` for comments on code. Possibly we could change the other single-line occurrences in this file as well if we wanted.